### PR TITLE
Feat: added trash, pinned, edit modal

### PR DESCRIPTION
**Pull request by vinc3nati**
- Trash
- Pinned Notes
- Rich Text Editor
- Edit Modal
image
image

### CODE REVIEW

File: src/components/NoteEditor/NoteEditor.jsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    className="input-area"
+    name="body"
+    required
+    autoComplete="off"
+    value={note.body}
+    onChange={handleChange}
+    placeholder={expand ? "Description" : "Click to add a note"}
+    onClick={activeNote}
+    onDoubleClick={hideNote}
+  /> */}
```
Comment (kotesh-arya) [inline src/components/NoteEditor/NoteEditor.jsx]: You can remove the un-used comment here, If you meant to keep it for understanding then a descriptive comment will be more helpful, as you were adding comments in the previous files 👍️

**Review by kotesh-arya — APPROVED** [review]
Just mentioned one suggestion , remaining looks good 👍🏻

**Review by PeeyushPrashant — APPROVED** [review]
looks good

File: src/components/NoteEditor/NoteEditor.jsx (hunk 1 above)
Comment (vinc3nati) [reply src/components/NoteEditor/NoteEditor.jsx]: sure

**Pull request merged**